docs/REST API/rest-api.md

```diff
@@ -4251,12 +4259,22 @@ This endpoint returns the list of custom MDM commands that have been executed.
 {
   "results": [
     {
-      "device_id": "145cafeb-87c7-4869-84d5-e4118a927746",
+      "host_uuid": "145cafeb-87c7-4869-84d5-e4118a927746",
       "command_uuid": "a2064cef-0000-1234-afb9-283e3c1d487e",
+      "platform": "darwin",
```

Comment:
Do we need to add platform now? I see the value but I think we can cut this and do it in a later story.

If `fleetctl get mdm commands` is going to use it, then that's a good argument to do it now.

If we already built it, then I think we can leave it in.

cc @mna

Reply:
@noahtalerman This is a different endpoint than the one I've started working on (mine is the platform-agnostic run command endpoint). I don't think anyone started work on this but I could be wrong (@gillespi314 maybe?).

Reply:
I removed `platform` field from the response @noahtalerman @mna